best-practices/README.md

```diff
@@ -13,6 +13,12 @@ General
 * [Keep the code simple](http://www.readability.com/~/ko2aqda2).
 * Avoid premature optimizations. Get a version 1.0 out there as soon as you
   can. Until you have some users to measure, you're optimizing based on guesses.
+* Use `bin/setup` for initial setup of the project and for development
+  environment seed data.
+* Use `bin/server` for starting all the processes required to run the project
+  in a development environment, when suitable.
+* Use `bin/deploy` for deploying the project to staging and production
```

Comment:
TBH, you haven't personally been working on several conferences / landing pages, which is the reason you haven't used `bin/deploy` much.

That said, I do agree that this rule is a bit meaningless nowadays, and we don't really need it.
The only projects that have a `bin/deploy` are the ones hosted on gh-pages (and maybe some older apps).
And given that there are so many variables (does the project have staging/production? do I need a password, and if so, where do I get it? what If I want to deploy a different branch?), this isn't as straightforward as a `setup` or a `server` to easily dump into a script

Also, for Heroku apps, I use `parity`

I would propose something like this instead:

> * Prefer CI deploys over local ones, and provide instructions on how they work (are they manually triggered or automatic? Is there a staging / review apps set up?)
> * If CI deploys aren't an option, provide clear instructions on how to handle them in your README